<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tab_Params</class>
 <widget class="QWidget" name="Tab_Params">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>529</width>
    <height>213</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Number of &amp;matrix molecules:</string>
     </property>
     <property name="buddy">
      <cstring>spin_numMatrixMols</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Number of &amp;simultaneous searches:</string>
     </property>
     <property name="buddy">
      <cstring>spin_numSearches</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Limits for IAD between matrix and &amp;center molecule:</string>
     </property>
     <property name="buddy">
      <cstring>spin_IAD_min</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Limits for matrix &amp;radius coordinate:</string>
     </property>
     <property name="buddy">
      <cstring>spin_radius_min</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSpinBox" name="spin_numSearches">
     <property name="maximum">
      <number>10000</number>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="spin_numMatrixMols"/>
   </item>
   <item row="4" column="1">
    <widget class="QDoubleSpinBox" name="spin_IAD_min"/>
   </item>
   <item row="4" column="2">
    <widget class="QDoubleSpinBox" name="spin_IAD_max"/>
   </item>
   <item row="5" column="1">
    <widget class="QDoubleSpinBox" name="spin_radius_min"/>
   </item>
   <item row="5" column="2">
    <widget class="QDoubleSpinBox" name="spin_radius_max"/>
   </item>
   <item row="8" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="3">
    <widget class="QCheckBox" name="cb_radius_auto">
     <property name="text">
      <string>Auto</string>
     </property>
    </widget>
   </item>
   <item row="8" column="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Final number of scenes:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="spin_cutoff">
     <property name="maximum">
      <number>10000</number>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QCheckBox" name="cb_cluster">
     <property name="toolTip">
      <string>If selected, a cluster of Matrix molecules will be built, optionally around the substrate molecule.

The matrix radius coordinate limits will be ignored, and the IAD range above will also be used to deterimine the distance between neighboring matrix molecules.</string>
     </property>
     <property name="text">
      <string>Build cluster</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QCheckBox" name="cb_strictHBonds">
     <property name="text">
      <string>Force hydrogen bonds</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QCheckBox" name="cb_build2DNetwork">
     <property name="text">
      <string>Build 2D Network</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>spin_numSearches</tabstop>
  <tabstop>spin_cutoff</tabstop>
  <tabstop>spin_numMatrixMols</tabstop>
  <tabstop>spin_IAD_min</tabstop>
  <tabstop>spin_IAD_max</tabstop>
  <tabstop>spin_radius_min</tabstop>
  <tabstop>spin_radius_max</tabstop>
  <tabstop>cb_radius_auto</tabstop>
  <tabstop>cb_cluster</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>cb_radius_auto</sender>
   <signal>clicked(bool)</signal>
   <receiver>spin_radius_min</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>826</x>
     <y>143</y>
    </hint>
    <hint type="destinationlabel">
     <x>526</x>
     <y>144</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cb_radius_auto</sender>
   <signal>clicked(bool)</signal>
   <receiver>spin_radius_max</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>943</x>
     <y>144</y>
    </hint>
    <hint type="destinationlabel">
     <x>661</x>
     <y>148</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
